### PR TITLE
Rewrite stopper code a bit, second try

### PR DESCRIPTION
**Pull request by heinrich5991**
Remove the whole copy-and-paste mess.

A seemingly ineffective and apparently code block has been removed.

The broken front layer from the first try has been fixed.

### CODE REVIEW

File: src/game/client/prediction/entities/character.cpp [hunk 1]
```diff
@@ -653,79 +649,30 @@ void CCharacter::HandleSkippableTiles(int Index)
...
-	m_TileFlags = Collision()->GetTileFlags(MapIndex);
-	m_TileIndexL = Collision()->GetTileIndex(MapIndexL);
-	m_TileFlagsL = Collision()->GetTileFlags(MapIndexL);
-	m_TileIndexR = Collision()->GetTileIndex(MapIndexR);
-	m_TileFlagsR = Collision()->GetTileFlags(MapIndexR);
-	m_TileIndexB = Collision()->GetTileIndex(MapIndexB);
-	m_TileFlagsB = Collision()->GetTileFlags(MapIndexB);
-	m_TileIndexT = Collision()->GetTileIndex(MapIndexT);
-	m_TileFlagsT = Collision()->GetTileFlags(MapIndexT);
 	m_TileFIndex = Collision()->GetFTileIndex(MapIndex);
```
Comment (heinrich5991) [inline src/game/client/prediction/entities/character.cpp]: This is the difference to last time btw, I had deleted this line and it made the front layer ineffective.

**Comment by fokkonaut** [comment]
Why do you us ClampVel instead of LimitVel?

**Comment by heinrich5991** [comment]
I use `LimitVel` in the places where `LimitForce` was used before (just a rename) and I use `ClampVel` where the crazy `if`s were used before. I tried to not change the logic of the code, that's why there are two different versions of stopper code.

**Comment by fokkonaut** [comment]
We could just merge those, cant we?

**Comment by heinrich5991** [comment]
@fokkonaut I tried merging them, but then I saw that I would need to make `m_MoveRestrictions` public to do that. I like it better with these two separate functions, I think.

**Comment by heinrich5991** [comment]
@12pm helpfully went through all the maps and removed nonfunctioning stoppers that were present in maps.

**Comment by heinrich5991** [comment]
I think this should be good to go now. @Learath2? ;)

**Comment by def-** [comment]
-200 lines, nice. I'd say let's merge.

**Review by Learath2 — APPROVED** [review]
Looks great to me. So much better.

**Pull request merged**